packages/docs/docs/getting-started/what-can-you-build-with-unirep.md

```diff
@@ -0,0 +1,29 @@
+
+Anonymous GitHub:
+- Some developers may prefer to publish code anonymously. In A GitHub-type application built with UniRep, the reputation values could be the amount of commits a developer makes a repository, the amount of stars they have collectively on their profile, and/or if they have made successful PRs on a project. These reputation values can then be used to grant access to the anonymous developer to push to the main branch.
+
+Eth or Token Attesters:
+- It is possible to create an attester where the reputation value is Eth or some Token. This would allow the transfer of value anonymously. 
+
+Multi-Attester Applications:
+- It is possible to build attesters that interact with each other. For instance, a social media application can have multiple different groups. Let's say group X has their own reputation value called "Doge" and group Y has their reputation value called "Elon". The rules of how to earn these reputation values are defined by the developer. Multi-attester applications can interact with each other in various ways, too. If a member from group Y earns 50 "Elon" then they could be granted access to another attester that must have at least 50 "Doge" to join. The rules become limitless at this point, but bear in mind all the ideas possible with UniRep; social media is only one example.
+
+Medical Records:
+- A problem in the Electronic Health Records industry is interoperability between different healthcare providers. If you go to Hospital X because you live in California, they will store your records in their system. If you go to Hospital Y in New York because you get seriously injured, they have no idea about your medical background because of poor sharing of records. With UniRep, a user could have their medical records be attested to them be each provider (each provider would be an attester in this case) and submit a zk-proof  that they are the owner of that data when a provider needs access to it. Aside from interoperability, this would also have the user always be in control of their records. Please note: this is only an example and we must assume new cryptography, like zero-knowledge proofs, could be broken in 50 years.
+
+Journalist Applicaitons:
+- Some countries have restrictive laws on speech. If a journalist is speaking freely and stating the trust, especially truth that is contrary to what their government wants to be known, there could be serious reprecussions for that individual. With UniRep, a journalist could prove that they are from a well regarded news organization in their country anonymously and have a badge from that organization when they report on their work. This creates trust anonymously.
```

Comment:
can UniRep can limit users from talking bad content because they are constrained by reputation from others.
There is a word called **attributable** (I don't know how to use it but you can include it in the paragraph 🤣)

Reply:
Or just say we can use Unirep to prevent spam but keep everyone anonymous